kajiki/template.py

```diff
@@ -408,9 +409,27 @@ def patch_code_file_lines(code, filename, firstlineno, lnotab):
                               lnotab,
                               code.co_freevars,
                               code.co_cellvars)
+elif version_info[1] < 8:
```

Comment:
make this a comparison between tuples: `sys.version_info < (3, 8)` it's more readable and explicit about the fact that it's 3.8.